### PR TITLE
packages.py:Changed few lines of code

### PR DESCRIPTION
**Pull request by shirishaganta**
Based on issue #1385 modified the script to run correctly

Signed-off-by: Shirisha Ganta <shiganta@in.ibm.com>

### CODE REVIEW

**Comment by shirishaganta** [comment]
JOB ID     : c48539c4d9df333a097eabf3833ce047de0add16
JOB LOG    : /home/tests/results/job-2019-11-25T08.57-c48539c/job.log
 (1/1) packages_test.py:Package_check.test: PASS (8.99 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 9.51 s
JOB HTML   : /home/tests/results/job-2019-11-25T08.57-c48539c/results.html

**Comment by shirishaganta** [comment]
@Pavithra1602  Please review this once

**Comment by Pavithra1602** [comment]
looks good

File: ras/packages.py [hunk 1]
```diff
@@ -41,15 +41,14 @@ def test(self):
                 'packages_rhel', default=['lshw', 'librtas'])
             self.packages.extend(packages_rhel)
             for package in self.packages:
-                if "anaconda" in process.system_output("yum list installed "
-                                                       "| grep %s | tail -1"
-                                                       % package,
-                                                       shell=True).decode("utf-8"):
+                if process.system_output("yum list installed "
```
Comment (harish-24) [inline ras/packages.py:44]: @shirishaganta Any reason why the following method is not used for all the distros?
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/1436/files#diff-75b317494162cabffcfe2bef4575ce52L69

I believe it must work on any given distro.

Comment (harish-24) [reply ras/packages.py:44]: @shirishaganta The above link does not seem to direct properly, I meant `smm.check_installed` here.. https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/ras/packages.py#L68

**Comment by shirishaganta** [comment]
I lost this branch on my TP. will discuss with pavithra and send a new patch for this if necessary because I have seen this testcase has passed without any changes . So closing it now

**Pull request closed without merging**